config/ur20/physical_parameters.yaml

```diff
@@ -34,47 +24,99 @@ inertia_parameters:
     base:
       radius: 0.075
       length: 0.038
-    shoulder:
-      radius: 0.075
-      length: 0.178
-    upperarm:
-      radius: 0.075
-      length: 0.852
-    forearm:
-      radius: 0.075
-      length: 0.7287
-    wrist_1:
-      radius: 0.075
-      length: 0.12
-    wrist_2:
-      radius: 0.075
-      length: 0.12
-    wrist_3:
-      radius: 0.045
-      length: 0.05
 
-  center_of_mass: # Adjusted manually
+  center_of_mass:
     shoulder_cog:
-      x: 0.0
-      y: 0.00244
-      z: -0.037
+      x: 0.0 # model.x
+      y: -0.0062 # -model.z
+      z: -0.061 # model.y
     upper_arm_cog:
-      x: 0.00001
-      y: 0.15061
-      z: 0.48757
+      x: -0.3394 # model.x - upperarm_length
+      y: 0.0 # model.y
+      z: 0.2098 # model.z
     forearm_cog:
-      x: -0.00012
-      y: 0.06112
-      z: 0.2984
+      x: -0.4053 # model.x - forearm_length
+      y: 0.0 # model.y
+      z: 0.0604 # model.z
     wrist_1_cog:
-      x: -0.00021
-      y: -0.00112
-      z: 0.02269
+      x: 0.0 # model.x
+      y: -0.0393 # -model.z
+      z: -0.0026 # model.y
     wrist_2_cog:
-      x: -0.00021
-      y: 0.00112
-      z: 0.002269
+      x: 0.0 # model.x
+      y: 0.0379 # model.z
+      z: -0.0024 # -model.y
     wrist_3_cog:
-      x: 0.0
-      y: -0.001156
-      z: -0.00149
+      x: 0.0 # model.x
+      y: 0.0 # model.y
+      z: -0.0293 # model.z
```

Comment:
The online documentation says different about this, as well. Your values match the values in URSim, though.